### PR TITLE
Update Discovery ISO modal

### PR DESCRIPTION
**Pull request by rawagner**
Screenshot from 2020-08-20 12-21-52

### CODE REVIEW

**Review by jtomasek — CHANGES_REQUESTED** [review]
I am getting
```Formik called `handleChange`, but you forgot to pass an `id` or `name` attribute to your input:```
when setting the ssh key via dragging or clicking the 'browse' button as well as when clicking 'clear'.

Looks pretty great btw.!

File: src/components/clusterConfiguration/DiscoveryImageForm.tsx [hunk 1]
```diff
@@ -29,6 +31,30 @@ import { updateCluster, forceReload } from '../../features/clusters/currentClust
 import { DiscoveryImageFormValues } from './types';
 import ProxyFields from './ProxyFields';
 
+import './discovery.css';
```
Comment (jtomasek) [inline src/components/clusterConfiguration/DiscoveryImageForm.tsx:34]:
```suggestion
import './discoveryImageModal.css';
```

maybe?

Comment (rawagner) [reply src/components/clusterConfiguration/DiscoveryImageForm.tsx]: renamed

File: src/components/clusterConfiguration/DiscoveryImageForm.tsx [hunk 2]
```diff
@@ -29,6 +31,30 @@ import { updateCluster, forceReload } from '../../features/clusters/currentClust
...
+type BootOrderProps = {
+  order: number;
+  Icon: typeof HddIcon;
+  title: string;
+  description: string;
+};
+
+const BootOrder: React.FC<BootOrderProps> = ({ order, Icon, title, description }) => (
+  <Text className="boot-order" component="p">
+    <Text className="boot-order-order" component={TextVariants.small}>
```
[the remark this answers is not in the document]
Comment (rawagner) [reply src/components/clusterConfiguration/DiscoveryImageForm.tsx]: fixed

File: src/components/clusterConfiguration/DiscoveryImageForm.tsx [hunk 3]
```diff
@@ -29,6 +31,30 @@ import { updateCluster, forceReload } from '../../features/clusters/currentClust
...
+
+type BootOrderProps = {
+  order: number;
+  Icon: typeof HddIcon;
+  title: string;
+  description: string;
+};
+
+const BootOrder: React.FC<BootOrderProps> = ({ order, Icon, title, description }) => (
+  <Text className="boot-order" component="p">
```
[the remark this answers is not in the document]
Comment (rawagner) [reply src/components/clusterConfiguration/DiscoveryImageForm.tsx]: Good idea, fixed

**Comment by rawagner** [comment]
@jtomasek file update errors should be fixed as well. PTAL

**Comment by andybraren** [comment]
It took me a minute to figure out if your screenshot was of the implementation or @nirfarkas' [design](https://marvelapp.com/prototype/78g662d/screen/71249234). 😄 This looks awesome!!

And it includes PatternFly's new file upload component!? ❤️

I see `Use HTTP Proxy` was changed to `Configure HTTPS Proxy` but that sounds fine to me too.

**Review by jtomasek — APPROVED** [review]

**Pull request merged**